tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_compose.py

```diff
@@ -0,0 +1,117 @@
+
+        self.docker_name = "-".join((config.get("name", "prplmesh_compose"), self.unique_id))
+        self.name = config.get("name", "prplmesh_compose")
+        print('config.get("name") {}'.format(config.get("name")))
+        # self.name = config.get("name")
+        self.role = config.get("role", "agent")
+        self.cleanup_cmd = config.get("cleanup_cmd", None)
+        self.conn_cmd = config.get("conn_cmd", None)
+        self.delay = config.get("delay", 7)
+        self.docker_network = "boardfarm-ci_default"
+
+        if self.role == "controller":
+            self._docker_compose(["-d", "--name", self.name, "controller"],
+                                 "run", "start-controller-agent")
+            time.sleep(self.delay)
+            self.controller_entity = \
+                ALEntityDocker(self.name, device=self, is_controller=True, compose=True)
+        else:
+            self._docker_compose(["-d", "--name", self.name, "agent"],
+                                 "run", "start-agent")
+            time.sleep(self.delay)
+            self.agent_entity = ALEntityDocker(self.name, device=self, is_controller=False, compose=True)
+
+        self.wired_sniffer = Sniffer(_get_bridge_interface(self.docker_network),
+                                     boardfarm.config.output_dir)
+        self.check_status()
+
+    def _docker_compose(self, args, parameter=None, start=None):
+        print('_docker_compose: args {}'.format(args))
+        yml_path = "tools/docker/boardfarm-ci/docker-compose.yml"
+        full_args = ["-f", os.path.join(rootdir, yml_path)]
+        if parameter == "run":
+            log_path = os.path.join(rootdir, "logs/{}".format(self.name))
+            if not os.path.exists(log_path):
+                os.mkdir(log_path)
+
+            pipeline_id = os.getenv('CI_PIPELINE_ID')
+            if pipeline_id is None or pipeline_id == 'latest':
+                vol = '{}:/tmp/{}/beerocks/logs'.format(log_path, self.user_id)
+            else:
+                vol = '{}:/tmp/beerocks/logs'.format(log_path)
+
+            full_args += ["run", "--rm", "-v", vol]
+            # full_args += ["--entrypoint", entrypoint + ' ' + start]
+            full_args += args
+
+        print('_docker_compose: {}'.format(' '.join(full_args)))
+        # os.environ['CURRENT_UID'] = '1000:998'
+        if os.getenv('CI_PIPELINE_ID') is None:
+            print('Setting CI_PIPELINE_ID "latest"')
+            os.environ['CI_PIPELINE_ID'] = 'latest'
+            self._run_shell_cmd("/usr/local/bin/docker-compose",
+                                full_args, env=os.environ)
+        else:
+            self._run_shell_cmd("/usr/local/bin/docker-compose", full_args)
+
+    def __del__(self):
+        # self._docker_compose(["stop", self.name])
+        self._run_shell_cmd("docker", ["stop", self.name])
+        self._run_shell_cmd("docker", ["container", "rm", "-f", self.name])
```

Comment:
This needs to use docker_name as well:
```suggestion
        self._run_shell_cmd("docker", ["stop", self.docker_name])
        self._run_shell_cmd("docker", ["container", "rm", "-f", self.docker_name])
```